src/SFA.DAS.QnA.Database/projects/roatp/sections/section1.3.json

```diff
@@ -80,19 +80,19 @@
             "SequenceId": null,
             "SectionId": null,
             "Title": "Enter your organisation's ultimate parent company details",
-            "LinkTitle": "",
+            "LinkTitle": "Enter your organisation's ultimate parent company details",
             "InfoText": "",
             "Questions": [
                 {
                     "QuestionId": "YO-21",
                     "QuestionTag": "Add-ParentCompanyNumber",
-                    "Label": "<span class='govuk-label'>Company number</span>",
+                    "Label": "Company number",
                     "ShortLabel": "",
-                    "QuestionBodyText": "",
+                    "QuestionBodyText": "<p class=\"govuk-body\">If you’re not sure, <a href=\"https://beta.companieshouse.gov.uk/\" target=\"_blank\">search for your UK ultimate parent company (opens in a new window or tab)</a>.</p>",
                     "Hint": "",
                     "Input": {
                         "Type": "Text",
-                        "InputClasses": "app-uppercase",
+                        "InputClasses": "govuk-input--width-10",
```

Comment:
Can the app-uppercase class be added back in, this activates JS code to enforce upper case alphanumeric in this field

Reply:
app-uppercase is not a standard gov InputClass (and is not supported by QnA Config)
I have asked @gdbarnes to findout what a govuk equivalent might be.